sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/extensions/checkpointstoretable/_tablestoragecs.py

```diff
@@ -22,17 +50,224 @@ class TableCheckpointStore:
+        :returns: A table checkpoint store.
+        :rtype: ~azure.eventhub.extensions.checkpointstoretable.TableCheckpointStore
+        """
+        endpoint, credential = parse_connection_str(
+            conn_str=conn_str, credential=None, keyword_args=kwargs
+        )
+        return cls(endpoint, table_name=table_name, credential=credential, **kwargs)
+
+    @classmethod
+    def _create_ownership_entity(cls, ownership):
+        """
+        create a dictionary with the new ownership attributes so that it can be updated in tables
+        """
+        ownership_entity = {
+            u'PartitionKey': "{} {} {} Ownership".format(ownership["fully_qualified_namespace"],
+        ownership["eventhub_name"], ownership['consumer_group']),
+            u'RowKey': ownership['partition_id'],
+            u'owner_id' : ownership['owner_id'],
+        }
+        return ownership_entity
+
+    @classmethod
+    def _create_checkpoint_entity(cls, checkpoint):
+        """
+        create a dictionary with the new checkpoint attributes so that it can be updated in tables
+        """
+        checkpoint_entity = {
+            u'PartitionKey': "{} {} {} Checkpoint".format(checkpoint["fully_qualified_namespace"],
+        checkpoint["eventhub_name"], checkpoint['consumer_group']),
+            u'RowKey': checkpoint['partition_id'],
+            u'offset' : checkpoint['offset'],
+            u'sequence_number' : checkpoint['sequence_number'],}
+        return checkpoint_entity
+
+    def list_ownership(self, fully_qualified_namespace, eventhub_name, consumer_group):
+        """Retrieves a complete ownership list from the storage table.
+        :param str fully_qualified_namespace: The fully qualified namespace that the Event Hub belongs to.
+         The format is like "<namespace>.servicebus.windows.net".
+        :param str eventhub_name: The name of the specific Event Hub the partition ownerships are associated with,
+         relative to the Event Hubs namespace that contains it.
+        :param str consumer_group: The name of the consumer group the ownerships are associated with.
+        :rtype: Iterable[Dict[str, Any]], Iterable of dictionaries containing partition ownership information:
+                - `fully_qualified_namespace` (str): The fully qualified namespace that the Event Hub belongs to.
+                  The format is like "<namespace>.servicebus.windows.net".
+                - `eventhub_name` (str): The name of the specific Event Hub the checkpoint is associated with,
+                  relative to the Event Hubs namespace that contains it.
+                - `consumer_group` (str): The name of the consumer group the ownership are associated with.
+                - `partition_id` (str): The partition ID which the checkpoint is created for.
+                - `owner_id` (str): A UUID representing the current owner of this partition.
+                - `last_modified_time` (float): The last time this ownership was claimed.
```

Comment:
azure-eventhub-checkpointstoreblob has this documented as a datetime.datetime, as does the `CheckpointStore` class. I just want to be 100% sure...was the azure-eventhub-checkpointstoreblob documentation incorrect and it actually returns a float as well? Or does it currently return a datetime.datetime?

Reply:
yes it is incorrect @swathipil and me ran the azure-eventhub-checkpointstoreblob and saw that last_modified_time was actually a float.